cmd/tetragon/main.go

```diff
@@ -372,6 +372,33 @@ func tetragonExecute() error {
 		if err != nil {
 			return err
 		}
+	} else {
```

Comment:
Please do not use an `else` here. Users should be able to load both from files and directories. (we should have the cli suporting multiple files and directories, but we can leave this as a followup). As I mentioned in another comment, please add an option for the directory.

Reply:
Ok, I agree here too... but just didn't want to change lot of things or behaviour at once, please see my reply above.  I can make it now read from /etc/tetragon/tetragon.tp.d/*  and --tracing-policy  if that's fine ?

Users should just clear out /etc/tetragon/tetragon.tp.d/ to restore

Reply:
I don't see why we would need a followup. This is a simple change. Let's just do it in this PR.